docs/content/reference/make-commands.md

```diff
@@ -55,14 +55,30 @@ git checkout -- . && git clean -f google/ google-beta/ website/
 
 For ease of contribution, we provide containers with the required dependencies for building magic-modules, as well as the option to build them yourself.
 
-[scripts/make-in-container.sh](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/scripts/make-in-container.sh) acts as a drop-in replacement for magic-modules `make` commands by setting up the containers and running `make` inside the container.
+[`./scripts/make-in-container.sh`](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/scripts/make-in-container.sh) acts as a drop-in replacement for magic-modules `make` commands by setting up the containers and running `make` inside the container. The script uses [Docker](https://docker.io/) if available and [Podman](https://podman.io/) otherwise. If you run into any problems, please [file an issue](https://github.com/hashicorp/terraform-provider-google/issues/new/choose).
```

Comment:
We should add a note to always run this command from the root of magic-modules. Alternatively we could try to detect the root of the directory by iterating up out of nested folder. This command will fail when run outside of root.